starfish/recipe/cli.py

```diff
@@ -0,0 +1,29 @@
+from slicedimage.io import resolve_path_or_url
+
+from starfish.config import StarfishConfig
+from starfish.starfish import starfish
+from starfish.util import click
+from .recipe import Recipe
+
+
+@starfish.command("recipe")
+@click.option("--recipe", required=True, type=str, metavar="RECIPE_PATH_OR_URL")
```

Comment:
I wonder why we have a public `run` method now that we can run recipes and you've shown this to be faster.

Reply:
For API users.